gocd/slfo-stagings.gocd.yaml.erb

```diff
@@ -274,8 +281,14 @@ pipelines:
                 if [ ${ret} -eq 2 ]; then
                     echo "product repository not found. Project configuration issue?" >&2
                     exit 1
+                elif [ ${ret} -ne 0 ]; then
```

Comment:
I think we can get rid of this and only keep the timeout check, as the exit status it's already checked before (line 280)